### PR TITLE
👨 ‍Create admin portal page and tabs

### PR DESCRIPTION
**Pull request by ayanmallik**
## Description

- Restructure navigation bar to hold all Admin actions under Admin tab
- Create subtabs within Admin page for Add Resource, Edit Home, Manage Users, and Manage Resources

Fixes #<number>

## Screenshots

image
image

### CODE REVIEW

File: client/src/components/ResourceForm.jsx [hunk 1]
```diff
@@ -160,11 +160,6 @@ const ResourceForm = (props: FormProps) => {
 
   return (
     <Layout className="resource-form-layout">
-      <Header className="header">
-        <Row justify="center" type="flex">
-          <h2>{id ? 'Edit a Resource' : 'Add a Resource'}</h2>
```
Comment (jackieo5023) [inline client/src/components/ResourceForm.jsx:165]: One tiny nit - can we add this logic to the "add resource" tab header?

**Review by jackieo5023 — APPROVED** [review]
Looks great Ayan! Thanks for pushing this out so fast :) I made one commit removing some tests that were failing since the routes didn't exist anymore - but feel free to merge after addressing my one comment (and getting a second review)!

**Review by kenetec — COMMENTED** [review]
this message has been approved by jackie™

**Review by nkonjeti — APPROVED** [review]
Great job!!

**Pull request merged**